packages/roosterjs-content-model-core/lib/coreApi/setDOMSelection.ts

```diff
@@ -44,6 +44,7 @@ export const setDOMSelection: SetDOMSelection = (core, selection, skipSelectionC
                 core.selection.selection = selection;
 
                 setRangeSelection(doc, image);
+                image.scrollIntoView();
```

Comment:
Can this work for docking scenario? 

We may need to do our own implementation to work together with getVisibleViewPort() function

Reply:
I checked the change and I do not see a particular problem with the scroll functionality in the Docking Plugin.

About implementing getVisibleViewPort, you mean that if we scroll to the element but it is still hidden by any sticky element in the docking functionality, we should still scroll the Image/Table into the visible viewport of the editor?

Reply:
yes. That is what I mean.

Reply:
Good point, I agree I will add this too.